tangelo/toolboxes/operators/tests/test_operators.py

```diff
@@ -39,6 +41,51 @@ def test_qubitop_to_qubitham(self):
         self.assertDictEqual(qubit_ham.__dict__, reference_attributes)
 
 
+class FermionOperatorTest(unittest.TestCase):
+
+    def test_get_coeff(self):
+        """Test the get_coeff method of the FermionOperator class."""
+
+        # Constant.
+        dummy_ferm_op = FermionOperator("", 1.)
+
+        # 1-body terms.
+        dummy_ferm_op += FermionOperator("0^ 0", 2.)
+        dummy_ferm_op += FermionOperator("0^ 1", 3.)
+        dummy_ferm_op += FermionOperator("1^ 0", 4.)
+        dummy_ferm_op += FermionOperator("1^ 1", 5.)
+
+        # 2-body term.
+        dummy_ferm_op += FermionOperator("0^ 0^ 0 0", 7.)
+        dummy_ferm_op += FermionOperator("1^ 1^ 1 1", 8.)
+        dummy_ferm_op += FermionOperator("0^ 1^ 0 1", 9.)
+
+        ref_cte = 1.
+        ref_one_body = np.array([
+            [2., 3.],
+            [4., 5.]
+        ])
+        ref_two_body = np.array([
```

Comment:
Alternative: define as `np.zeros((2,2,2,2))` and set up the 3 only non-zero values explicitly.